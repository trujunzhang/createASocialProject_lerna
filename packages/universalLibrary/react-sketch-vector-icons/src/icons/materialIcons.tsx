import * as React from 'react'
import { View, ViewProps, ViewStyle, Text, TextProps } from 'react-primitives'
import { IconProps } from 'react-native-vector-icons/Icon'

interface IMaterialIconsProps {}
interface IMaterialIconsState {}

export class MaterialIcons extends React.Component<IconProps, any> {
  render() {
    return <Text>{name + 'wh-MaterialIcons'}</Text>
  }
}
