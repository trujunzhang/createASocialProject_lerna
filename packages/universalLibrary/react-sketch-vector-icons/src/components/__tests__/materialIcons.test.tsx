import * as React from 'react' // So important
import { MaterialIcons } from '../materialIcons'

import * as renderer from 'react-test-renderer'

it('renders without crashing', () => {
  const page = <MaterialIcons name="get-app" size={32} style={{ color: '#999' }} />
  const rendered = renderer.create(page).toJSON()
  expect(rendered).toMatchSnapshot()
})
