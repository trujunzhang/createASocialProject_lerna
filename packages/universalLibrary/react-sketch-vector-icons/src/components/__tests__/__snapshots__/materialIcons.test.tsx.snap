// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<svg
  fill="currentColor"
  height={32}
  name="MaterialIcons-[get-app]"
  preserveAspectRatio="xMidYMid meet"
  size={32}
  stroke="currentColor"
  strokeWidth="0"
  style={
    Object {
      "color": undefined,
    }
  }
  viewBox="0 0 24 24"
  width={32}
  xmlns="http://www.w3.org/2000/svg"
>
  <svg_path
    d="M248 167.5l64.9 98.8H183.1l64.9-98.8zM496 256c0 136.9-111.1 248-248 248S0 392.9 0 256 111.1 8 248 8s248 111.1 248 248zm-99.8 82.7L248 115.5 99.8 338.7h30.4l33.6-51.7h168.6l33.6 51.7h30.2z"
  />
</svg>
`;
